app/src/main/java/com/duckduckgo/app/bookmarks/ui/bookmarkfolders/BookmarkFoldersAdapter.kt

```diff
@@ -113,41 +112,44 @@ sealed class BookmarkFolderScreenViewHolders(itemView: View) : RecyclerView.View
 
     class BookmarkFoldersViewHolder(
         private val layoutInflater: LayoutInflater,
-        private val binding: ViewBookmarkFolderEntryBinding,
+        private val binding: RowTwoLineItemBinding,
         private val viewModel: BookmarksViewModel,
     ) : BookmarkFolderScreenViewHolders(binding.root) {
 
         private val context: Context = binding.root.context
 
         fun update(bookmarkFolder: BookmarkFolder) {
-            binding.overflowMenu.contentDescription = context.getString(
-                R.string.bookmarkOverflowContentDescription,
-                bookmarkFolder.name,
-            )
+            val listItem = binding.root
+            // fixme: After migration this contentDescription will be lost
```

Comment:
is this needed?

Reply:
This is a fixme note as I think we'll need to add a method to the component to set ContentDescription at some point. We are missing this one because the migration. Not a priority just now but I would leave it as a reminder